src/reducers/index.js

```diff
@@ -27,8 +27,8 @@ const cartItems = (state = [], action) => {
       return newState;
     case 'REMOVE_FROM_CART':
       return _.filter(state, (item) => item !== action.cartItem);
```

Comment:
_.filter is actually a lodash function that is not useful since 
most of the browsers Array type has now its own native .filter function.

(and for Interner Explorer, it is just needed to add a 'require('babel-polyfill') at the beginning of the application for having it)

So I would write it simpler here:
```
case 'REMOVE_FROM_CART':
    return state.filter(item => item !== action.cartItem)
```
note you can remove the bracket inside of the anonymous arrow function, since you need just the first arg

Reply:
Ok
Polyfill is already loaded [here](https://github.com/coopcycle/coopcycle-js/blob/d1aa9177bbca015102f60ed3df0fa795c611bcfe/webpack/config.js#L7) actually